ViewTemplates/Main/dashboard.blade.php

```diff
@@ -96,7 +96,7 @@ class="alert alert-danger col col-md-6 col-xl-12"
 
         <div v-if="error === null"
              class="col col-md-6 col-xl-12 mb-4 d-flex flex-row align-items-center gap-1">
-            <div class="progress flex-grow-1" role="progressbar" style="height: 2em"
+            <div class="progress flex-grow-1 h-100" role="progressbar"
```

Comment:
yes - i considered it close enough

Reply:
Looking at thyis again I actually prefer the h-100 as it makes the bar and the button the same height instead of the bar being slighlty shorter but off center alignment.

![image](https://github.com/akeeba/panopticon/assets/1296369/6d530e89-1537-4d6c-85e5-0287dddbdbf6)

![image](https://github.com/akeeba/panopticon/assets/1296369/e0d5ec87-bb40-43ed-b494-989d8740d507)

Alternatively if I change this to h-75 then it is only 1px shorter than 2em

Reply:
just let me know what you prefer and then this can get merged and I can finish the other inline stuff etc

Reply:
I don't like `h-100`. It looks very chonky. `h-75` looks more balanced to me; that was the reason I had the inline 2em height.

So, let's go with `h-75`.

Reply:
75 it is then